File: BackEnd/CorpsInventoryManagementSystem/Model/Domain/Category.cs
namespace CorpsInventoryManagementSystem.Model.Domain
{
    public class Category
    {
        public int Id { get; set; }
        public string CategoryId { get; set; }
        public string CategoryName { get; set; }
        public string Description { get; set; }
        public bool IsActivated { get; set; }  

        //Navigation Property
        //public ICollection<Product> Product { get; set; }
    }
}

File: FrontEnd/InventorySystem/src/app/model/category.ts
export interface ICategory {
    id : number,
    categoryId : string,
    categoryName: string,
    description: string,
    isActivated: boolean
}

export interface IAddNewCategory
{
    categoryId  : string,
    categoryName: string,
    description: string,
    isActivated: boolean
}

export interface IUpdateCategory
{
    categoryId : string,
    categoryName : string,
    description : string,
    isActivated : boolean
}
